server/graphql/v2/query/collection/OrdersCollectionQuery.ts

```diff
@@ -598,6 +619,33 @@ export const OrdersCollectionResolver = async (args, req: express.Request) => {
     totalCount: () => models.Order.count({ include, where }),
     limit: args.limit,
     offset: args.offset,
+    createdByUsers: async () => {
+      const userIds = await models.Order.findAll({
+        attributes: [[sequelize.fn('DISTINCT', sequelize.col('Order.CreatedByUserId')), 'CreatedByUserId']],
+        where: baseWhere,
+        include: baseInclude,
+        raw: true,
+      }).then(results => results.map(r => r.CreatedByUserId).filter(id => id !== null));
+
+      if (userIds.length === 0) {
+        return [];
+      }
+
+      const users = await models.User.findAll({
+        attributes: ['CollectiveId'],
+        where: { id: { [Op.in]: userIds } },
+        raw: true,
+      });
+
+      const collectiveIds = users.map(u => u.CollectiveId).filter(id => id !== null);
+      if (collectiveIds.length === 0) {
+        return [];
+      }
+
+      return models.Collective.findAll({
+        where: { id: { [Op.in]: collectiveIds } },
+      });
+    },
```

Comment:
This solution will not work from a performance perspective.

I you take a host like OSC, the number of individual users for contributions is around ~150.000:

```sql
select COUNT(DISTINCT o."CreatedByUserId")
from "Orders" o
INNER JOIN "Collectives" c ON o."CollectiveId" = c.id
WHERE c."HostCollectiveId" = 11004
AND o."deletedAt" IS NULL
```

Loading 150.000 profiles means taking hundreds of MB in the server RAM, and that's without speaking about data transfer or SQL cost.

The proper solution must be:
1. Paginated: only return the proper subset
2. Searchable: API returns entries based on the typed search term. I doubt that filtering a list of 150.000 entries with frontend JS would provide a responsive UI.